atlp-rwanda/commanders-rn-medica
# Feature: Updating and cancelling appointment

### What does this pr do ?
This practice is about when a patient want to update or cancel his or her appointment

**Required in this pr**
1. First, login to your account and update your account in order to be patient
2. Book any appointment if you have not already booked 

 **How to view this pr** 
After logging in, go to your appointment. You will see your appointment booked; if not, go to the doctor to view her or his appointment and book an appointment. so in the appointment, you see button for cancel and reschedule appointments, and then start to cancel or reschedule according to your choice

**How to test this**
1. If you finish canceling an appointment, you see it removed in upcoming and it is going to cancelled
2. If you finish rescheduling an appointment, you will see that the date and time can be changed on appointment found in upcoming
